ci/zephyr_run.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -x
+
+export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
+export ZEPHYR_SDK_INSTALL_DIR=$HOME/zephyr-sdk
+export ZEPHYR_BASE=$HOME/zephyr
+
+mkdir zephyr-build && pushd zephyr-build
+
+cmake -DBOARD=qemu_x86 $ZEPHYR_BASE/tests/subsys/fs/nffs_fs_api/basic
```

Comment:
Can you enable other nffs test suites as well?

Reply:
I have not tested the others but nothing is stopping adding support for them I guess...